src/lib/entity-state.ts

```diff
@@ -486,10 +491,18 @@ export abstract class EntityState<T> {
     return entities;
   }
 
-  private setup(storeClass: Type<EntityState<T>>, ...actions: string[]) {
+  private setup(storeClass: Type<EntityState<T>>, actions: string[]) {
+    const baseProto = Reflect.getPrototypeOf(storeClass.prototype);
```

Comment:
Why not `Object.getPrototypeOf`?

Reply:
Ping @markwhitfield

Reply:
Using `Reflect.getPrototypeOf` is the recommended way since ES6 became standard. One of the goals of ES6 is to provide alternative places for the methods on the already polluted `Object` class and global namespace. Some examples of these are `Reflect`, `Number`, etc.
Not a deal breaker though because both work the same.